script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -544,136 +549,131 @@ Excluding those, the survival rate is probably around 10%.
+I'll pour every last one of my efforts into these, and drag that 0% chance up to 1%.
 }
 [sha:e2e238124fb74e24181b1d734a5a61a6278b5702]{
 -- Page 693, Offset 10526.
 -- この二つの<致|マ><死|イ><地|ナ><点|ス>を踏み越えるため、意識を固める。
-In order to overcome those two fatal scenarios, I steel myself.
+I steel myself in order to overcome these two fatal scenarios.
 }
 [sha:edb2184f7c371fc63c68e09ea4e77e3040765489]{
 -- Page 693, Offset 10527.
 -- それ以外の事象に振り分ける<意識|よぶん>はない。
-I have no mental capacity left to allocate to any other eventualities.
+There is nothing left in my consciousness to allocate to any other eventualities.
 }
 [sha:0df06b142239c91ae758079373e614b3eefa6017]{
 -- Page 693, Offset 10528.
 -- だからもう、降り始めた俺には<対|も><応|の>を考える機能はない。
-So now that I've started to descend,
- I don't have the ability to think about how to respond to him.
+Now that I've initiated the descent,
+ I no longer have the mental capacity to react to his actions.
 }
 [sha:c057faa764c9a44cf6a342af0afa3732afa36dc0]{
 -- Page 694, Offset 10529.
 -- もしマイナスが二ヵ所に留まらない場合は？
-// "in that period" added to clarify that the problem is his clock-rate and the
-// limited time period
-What if I have to handle more than just two of those do-or-die moments in that period?
+What if there are more than two of these critical junctures?
 }
 [sha:a246cbce56656d812d3d22aa8c69c334bdbeef13]{
 -- Page 694, Offset 10530.
 -- それは考慮しない。三ヵ所もの『死』を超える余力は俺にはない。その時はヤツの勝ちだ。ヤツの異常性が、俺の想像を上回っているだけの話。
-I do not consider that.
- I don't have the capacity to overcome three 'deaths' in my mind.
+I won't consider it.
+ I don't have it in me to overcome three of them.
  If that's how it plays out, victory is his.
- It would just mean that his power is stronger than I can imagine.
+ That would just mean his powers were stronger than I imagined.
 }
 [sha:75680ba2b152fc7b3c04412eff6c4bb7e01ba93f]{
 -- Page 695, Offset 10531.
 -- 氷の槍は折り重なるように、落下する獲物に殺到する。
-A dense flurry of ice spears shoots toward the falling prey.
+A dense flurry of ice spears bombard the falling prey.
 }
 [sha:ba9c2a1086ba55450f0e1c97fe70b959fa0c1def]{
 -- Page 695, Offset 10532.
 -- まるで獣の<顎|あぎと>だ。恐ろしいがどうでもいい。
-It's like the jaws of a beast. It's terrifying, but it doesn't matter.
+Not unlike the jaws of a beast. Terrifying, but inconsequential.
 }
 [sha:9f532dbc35cebc5dc45cc94b45e9300a64065ff8]{
 -- Page 695, Offset 10533.
 -- そんな、ただ本能に因っただけの<咀嚼|そしゃく>など知った事か。
-I wonder if it's thanks to his instincts that he anticipated this.
+You really think you'll be able to devour me like that? Driven by nothing but instinct?
 }
 [sha:0032cdf383b9b0e7c3f16d1068f5192c946dc6f9]{
 -- Page 695, Offset 10534.
 -- 俺がそうくると確信し、そうなれば殺されると恐れた迎撃は、
-Convinced that I would come for him,
- and terrified that I would kill him when I did...
+No. That isn't the counterattack I feared was coming――the one I feared would end my life.
 }
 [sha:a7dd57312c121a7f9c51ed5e3e188e8021d13109]{
 -- Page 696, Offset 10535.
 -- ―――10メートル。
-―――Ten meters.
+At 10 meters...
```

Comment:
Still prefer this as a hard statement, either 
> Ten meters [left].

or 
> ―――Ten meters.

either way, ten should be spelled out